plugin/manager_linux.go

```diff
@@ -183,8 +183,8 @@ func (pm *Manager) upgradePlugin(p *v2.Plugin, configDigest digest.Digest, blobs
 				return
 			}
 
-			if err := os.Rename(backup, orig); err != nil {
-				err = errors.Wrap(err, "error restoring old plugin root on upgrade failure")
```

Comment:
Yup, I can change that; I decided to "log" to keep the old behavior, but if we need to error out, I'll return the error instead